CsCore/PlainNetClassLib/src/Plugins/CsCore/com/csutil/algorithms/AaaaImageCompareAaaa.cs

```diff
@@ -0,0 +1,107 @@
+﻿using StbImageSharp;
+using System;
+using System.Drawing;
+using System.Numerics;
+
+namespace com.csutil.src.Plugins.CsCore.com.csutil.algorithms {
+
+    public class AaaaImageCompareAaaa {
+
+        public AaaaImageCompareAaaa() { }
+
+        public CompareResult ImageCompare(ImageResult img1, ImageResult img2, double redScale=5) {
+
+            CompareResult result = new CompareResult();
+
+            ImageResult resultImage = new ImageResult();
+            resultImage.Data = img1.Data.DeepCopy();
```

Comment:
A note: DeepCopy() uses json serialization so might be much slower vs you just copying the image data (which is a byte array I think?) yourself